package.json

```diff
@@ -219,5 +246,13 @@
     "type": "opencollective",
     "url": "https://opencollective.com/verdaccio",
     "logo": "https://opencollective.com/verdaccio/logo.txt"
+  },
+  "dependenciesMeta": {
```

Comment:
Please why this change is necessary?

Reply:
Only if we use `pnp`, so in this case are not need it, but this command https://yarnpkg.com/cli/unplug#gatsby-focus-wrapper actually create the new `dependenciesMeta`, is a new field introduced by yarn. 

I'll remove it for now.